knative-operator/Gopkg.toml

```diff
@@ -1,6 +1,11 @@
-[[override]]
-  name = "github.com/go-openapi/spec"
-  branch = "master"
+[prune]
```

Comment:
Looks like that the file has been restructured as well (i.e moving this section to the top), so it's not easy to me to find out the real difference.

What actually is the reasoning for this reordering ?

Reply:
To group things mostly so it's clearer why certain deps are pinned (because they're usually in relation to each other)